### DISCUSSION TITLE
Override section in convention

### DISCUSSION THREAD

**Discussion by somenergia-it** [pawamoy/git-changelog#94] [Q&A]
What's a simple way to tell `git-changelog` to treat `new` as equivalent to `feat` during parsing, for example? If creating a new convention is the only way, how do I go about it? 

Thanks for such a great library!

**Comment by pawamoy** [comment]
Thanks!

Not possible yet, but there are two pending PRs that would allow creating new/custom conventions:

- https://github.com/pawamoy/git-changelog/pull/56
- https://github.com/pawamoy/git-changelog/pull/60
